src/docs/files/devguide05.md

```diff
@@ -20,54 +22,68 @@
-- On the right hand side you can see the parameters you need to fill in to place an order in the auction.
+2) This parameter represents the token that the auctioneer has selected to be the one accepted for bidding in the auction. This is the asset you need to bid with. 
 
-<img src="/assets/GA7.png" alt="drawing" width="500"/>
+3) This parameter shows a countdown of the duration of the auction. This parameter marks when the auction will end.
 
-8. In the amount section you would need to input the amount of bidding tokens you are willing to bid in the auction. You can either select the amount you want to bid or directly use the max button to bid all the tokens you have.
+4) This parameter shows the closing price of the auction if no more bids are submitted or canceled. As a bidder, you simply need to bid above this price for higher possibility to place a successful bid. 
 
-<img src="/assets/GA8.png" alt="drawing" width="500"/>
+5) This parameter is the Starting price of token auctioned. The auctioneer has decided its value and it establishes the fact that bidders can’t bid under this price.
 
-9. In the price per bid-token/auction token, the dapp sets by default the price that the auctioneers have set to be the minimum price. From that starting price, you can decide what is the price per bid-token/auction token you are willing to offer.
+6) The amount parameter, is the total amount of bidding tokens you are willing to pay for the auctioneer tokens.
 
-<img src="/assets/GA9.png" alt="drawing" width="500"/>
- 
-10. If it is the first time that you have used that token to bid, you will have to approve the contract to spend that token. First click on the "Approve" button, and then confirm the transaction on Metamask, as shown below.
+​	(a) you would need to input the amount of bidding tokens you are willing to bid in the auction. You can either select 		the amount you want to bid or directly use the max button to bid all the tokens you have.
 
-<img src="/assets/GA10.png" alt="drawing" width="500"/>
+7) The price per bid-token/auction token parameter is the max bid price you are willing to offer per token auctioned. 
 
-<img src="/assets/GA11.png" alt="drawing" width="500"/>
+​	(a) The dapp sets by default the price that the auctioneers have set to be the minimum price. From that starting price, 		you can decide what is the price per bid-token/auction token you are willing to offer.
 
-11. After you have set both the amount of tokens to bid, and the price per bid-token/auction token, you no longer see the enter amount section, instead you see a colored bar with the words “Place Order”. Once you are sure about the parameters you have inputted you have to click on “place order” to include your bid in the auction.
+8) The white line shows the current price, which will be the closing price if no more bids are submitted or canceled. And will dynamically move with new bids.
```

Comment:
Thie white dashed line indicates the current price. Also it would be the auction's closing price assuming no more orders would be placed or canceled

Reply:
Modified